src/components/App/index.js

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
+import { Provider } from 'react-redux';
+import { Route } from 'react-router-dom';
+import { ConnectedRouter } from 'connected-react-router';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import { withStyles } from '@material-ui/core/styles';
+import { host } from '../../config';
+import NavBar from '../common/NavBar/index';
+import Login from '../Login/index';
+import Typography from '@material-ui/core/Typography';
+import WalletsPage from '../Wallet/WalletsPage';
+import { MuiThemeProvider, createMuiTheme } from '@material-ui/core/styles';
+import { store } from '../../store';
+import { history } from '../../history';
+import './styles.css';
```

Comment:
should we migrate to sass ?

Reply:
i don't think so, because material-ui lib propose using style like this:
```
const styles = theme => ({
  root: {
    display: 'flex',
  }
});
```
which injects to component with HOC:
```
withStyles(styles)(Index);
```